events/filters.py

```diff
@@ -11,10 +12,13 @@ class OccurrenceFilter(django_filters.FilterSet):
     upcoming = django_filters.BooleanFilter(
         method="filter_by_upcoming", field_name="time"
     )
+    venue = django_filters.CharFilter(
```

Comment:
would `venue_id` be better name for this? to make it more descriptive 🙂 OTOH, `venue` could work too. your call.

Reply:
That'd make sense +1. I'll update it and the qs params below